akka-remote/src/main/scala/akka/remote/artery/TaskRunner.scala

```diff
@@ -101,7 +101,7 @@ private[akka] object TaskRunner {
       val yielding = 5 * idleCpuLevel
       val minParkNanos = 1
       // park between 250 and 10 micros depending on idleCpuLevel
-      val maxParkNanos = MICROSECONDS.toNanos(280 - 30 * idleCpuLevel)
+      val maxParkNanos = MICROSECONDS.toNanos(250 - (80 * (idleCpuLevel - 1) / 3))
```

Comment:
Oops, I was blind!

Hmm, however, this makes `maxParkNanos` between 220 and 10,  `spinning` between 1200 and 8900. Not consistent with the comments. I guess we should change either the comments or the calculation.